### ISSUE TITLE
Move print buttons to accessibility menu

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#1431]
Make Tools category
Alternate Formats:
- [x] Print
- [x] Speed Reader
- [x] PDF #280 , specifically https://github.com/elmsln/elmsln/issues/280#issuecomment-164731998 via whtml library

These are all basically alternate formats of the same thing. They already should have access to print permissions (need to check) but it's just not visible where to do that currently.

**Comment by btopro** [comment]
all there now!

**Comment by btopro** [comment]
![2016-11-01_21-02-02](https://cloud.githubusercontent.com/assets/329735/19913170/7f2e2e2a-a076-11e6-817e-98286fb5bcfc.png)
